src/lambda/explorer/standaloneNodes.ts

```diff
@@ -39,13 +40,15 @@ export class DefaultStandaloneFunctionGroupNode extends AWSTreeNodeBase implemen
         this.functionNodes = new Map<string, StandaloneFunctionNode>()
     }
 
-    public async getChildren(): Promise<StandaloneFunctionNode[]> {
-        await this.updateChildren()
+    public async getChildren(): Promise<(StandaloneFunctionNode | ErrorNode)[]> {
+        await this.handleErrorProneOperation()
 
-        return [...this.functionNodes.values()]
+        return this.errorNode ? [this.errorNode]
```

Comment:
Everywhere you have something like this, please check `!!this.errorNode` instead of `this.errorNode`
See https://medium.com/@pddivine/javascript-bang-bang-i-shot-you-down-use-of-double-bangs-in-javascript-7c9d94446054